packages/@okta/vuepress-site/docs/concepts/brands/index.md

```diff
@@ -113,6 +129,12 @@ Keep in mind the following when setting up domains for an org with branded email
 
 The main Okta email provider allows you to use each unique email sender (root domain, no-reply@company.com) only once in each cell. To bypass this limitation, use a subdomain to keep email senders unique (for example, no-reply@brandA.company.com and no-reply@brandB.company.com).
 
+#### User Activation email
+
+Use the [Brands API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Customization/#tag/Customization/operation/createEmailCustomization) to ensure that your customers receive the User Activation email with the correct branding. If you use the Admin Console, the default Okta branding is applied to the User Activation email. Otherwise, Okta applies the branding based on the domain used with each brand. See [Multibrand and custom domains](#multibrand-and-custom-domains).
```

Comment:
Should this call out the specific operation instead of just `Brands API`?

Something like `Use the Create an Email Customization operation to....`

Reply:
Also, slightly connected, should we change the tag on OAS3 from `Customizations` to `Brands`, or is it OK as is?

Reply:
@isaacdurham-okta Good point. I'm actually waiting to hear back from the dev team as to which endpoint or endpoints they'd use.

As for the tag on OAS3, it was a PM decision to start using "Customizations" to apply to what the Brands API does. They'd like to keep it as is.

Reply:
I think this content should start with explaining behavior for Admin Console, followed by
"If User Activation email with branding is expected, then use the [Activation API] with appropriate custom domain to activate the users..."